Add release notes for subctl PR #978 (#1075)
submariner-io/submariner-website: src/content/community/releases/_index.en.md

## v0.17.0

### New features

> * Added a new option, invoked via `--only basic-connectivity`, to `subctl verify` that runs a smaller set of connectivity tests as a quick
>   sanity check when time is a constraint.

Reviewer: ```suggestion
* The new `--only basic-connectivity` option on `subctl verify` runs a smaller set of connectivity tests as a quick
  sanity check when time is a constraint.
```
